internal/artifacts/validate_test.go

```diff
@@ -46,7 +46,7 @@ var _ = Describe("Validate", func() {
 		})
 		When("sending extension files", func() {
 			It("succeeds when the files are valid", func() {
-				Ω(ExecuteValidation(getTestPath("mta_with_ext"), "dev", []string{"cf-mtaext.mtaext", "other.mtaext"}, "semantic", "true", "", os.Getwd)).Should(Succeed())
+				Ω(ExecuteValidation(getTestPath("mta_with_ext"), "dev", []string{"cf-mtaext.mtaext", "other.mtaext"}, "semantic", "true", "emptyPath", os.Getwd)).Should(Succeed())
```

Comment:
Instead of changing the test I think you should change the mta.yaml in mta_with_ext directory (since this is supposed to be an example of a valid file)

Reply:
changed